packages/styled-components/src/constructors/constructWithOptions.ts

```diff
@@ -50,6 +50,10 @@ export interface Styled<
     : object,
   OuterStatics extends object = object
 > {
+  (
+    initialStyles: Styles<OuterProps>,
+    ...interpolations: Interpolation<OuterProps>[]
+  ): IStyledComponent<R, Target, OuterProps> & OuterStatics;
```

Comment:
In my experimentation, this overload might not be part of the problem. Did you need it?